_includes/footer_scripts.html

```diff
@@ -17,5 +17,4 @@
     window._ie = window._ie9 || window._ie10 || window._ie11;
   </script>
 
-  <script src="{{ site.baseurl }}/js/vendor/uswds.v0.9.6.js"></script>
-  <script src="{{ site.baseurl }}/js/index.js"></script>
+  <script src="{{ site.baseurl }}/assets/bundle.js"></script>
```

Comment:
Since we support HTTP/2 (which supports multiplexing), what's the advantage of packing these assets into one file?